src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --primary-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+}
+
+.logo-container a {
+  width: 40px;
+  height: 40px;
+}
+
+.header-nav-list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.header-nav-link {
+  margin-left: 20px;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  align-items: center;
+  display: flex;
+  height: 60px;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could use `font-family: Roboto, Arial, sans-serif;`.